api/app/src/mappings/garmin/activity-types.ts

```diff
@@ -1,6 +1,23 @@
 import { capitalize } from "lodash";
-import z from "zod";
+import { z } from "zod";
 
+function containsStringIndex(map: Record<ActivityType, string>, index: string): boolean {
+  return Object.prototype.hasOwnProperty.call(map, index);
+}
+
+export function activityTypeReadable(activityType: string): string {
+  // will the activity type mapping if it exists, and otherwise for example will change strings from:
+  //    - "SWIM_TO_BIKE_TRANSITION_V2" to "Swim To Bike Transition"
+  //    - "CROSS_COUNTRY_SKIING_WS" to "Cross Country Skiing"
+  const stringToUse = containsStringIndex(activityTypeMapping, activityType)
+    ? activityTypeMapping[activityType as ActivityType]
+    : activityType.replace(/_|V\d+|WS/g, " ");
+  return capitalize(stringToUse).trim();
+}
```

Comment:
This is a great candidate for some unit tests! 🤓

Reply:
I had a feeling this was going to be a comment 😆 

Will add some tests